napps/kytos/of_l2ls/main.py

```diff
@@ -51,6 +52,9 @@ def handle_packet_in(self, event):
 
         switch.update_mac_table(ethernet.source, in_port)
 
+        if packet_in.reason != PacketInReason.OFPR_NO_MATCH:
+            return
```

Comment:
For now, if your second napp send a action "send to controller" is your napp rule to handle this. So for now, let's keep as on this PR.